src/e2e/java/uk.gov.hmcts.reform.sscs/functional/ProcessCaseFile.java

```diff
@@ -27,23 +27,33 @@
 
     private final String caseloaderinstance = System.getenv("TEST_URL");
     String filename;
-    String outputdir = "src/test/resources/updates";
-
+    private static final String outputdir = "src/test/resources/updates";
+    private static final String processedReference =
+        "/incoming/processed/SSCS_Extract_Reference_2018-02-13-20-09-33.xml";
```

Comment:
In case of Reference file change it will fail and we will start experiencing same issue again. Can't it be resolved by variable to define Reference file, or find to process all "*reference*.xml" files instead (no case sensitive)?

Reply:
@iguannago -  As discussed with you earlier we will always use same reference file in preview and AAT and we will look into later generating this reference file also. are you happy with above approach if not let me know

Reply:
Ok approving this for now, but hardcoding isn't correct as mentioned especially when it is easily to resolve